scheduler/scheduler.go

```diff
@@ -167,8 +142,13 @@ func (s *eremeticScheduler) StatusUpdate(driver sched.SchedulerDriver, status *m
 		Time:   time.Now().Unix(),
 	})
 
+	if *status.State == mesos.TaskState_TASK_RUNNING {
+		TasksRunning.Inc()
```

Comment:
Will this not potentially create false positives due to receiving multiple task_running statuses? Might be a corner case, but still.